pkgs/top-level/python-packages.nix

```
@@ -4498,7 +4498,7 @@ in {
 
   playsound = callPackage ../development/python-modules/playsound { };
 
-  plexapi = callPackage ../development/python-modules/plexapi { };
+  plexapi = disabledIf isPy27 (callPackage ../development/python-modules/plexapi { });
```

please move into expression
```suggestion
  plexapi = callPackage ../development/python-modules/plexapi { };
```

---

Done. Thanks for the information :+1: